2021/01/29 11:32:02 : products_errors 9: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

2021/01/29 11:32:02 : products_errors 10: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

2021/01/29 11:32:02 : products_errors 16: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

2021/01/29 11:32:02 : products_errors 17: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

2021/01/29 11:32:03 : products_errors 18: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

2021/01/29 11:32:03 : products_errors 19: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

2021/01/29 11:32:03 : products_errors 20: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

2021/01/29 11:32:03 : products_errors 22: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

2021/01/29 11:32:03 : products_errors 23: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

2021/01/29 11:32:03 : products_errors 24: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

2021/01/29 11:32:03 : products_errors 25: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

2021/01/29 11:32:03 : products_errors 37: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

2021/01/29 11:32:03 : products_errors 39: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

2021/01/29 11:32:03 : products_errors 41: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

2021/01/29 11:32:03 : products_errors 42: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

2021/01/29 11:32:03 : products_errors 43: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

2021/01/29 11:32:04 : products_errors 44: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

2021/01/29 11:32:04 : products_errors 47: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

2021/01/29 11:32:04 : products_errors 48: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

2021/01/29 11:32:04 : products_errors 49: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1063, in migration
    check_import = fn_check_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2781, in check_product_import
    id_src=attr['value_id'],
KeyError: 'value_id'

